client/src/styles/PortfolioTable.scss

```diff
@@ -0,0 +1,220 @@
+      margin-left: 0.5rem;
+      span {
+        // This small adjustment aligns second arrow in icon with first
+        display: inline-block;
+        transform: translateY(0.5px);
+      }
+    }
+
+    .col-address {
+      position: sticky;
+      left: 0px;
+      z-index: 40;
+      background-color: inherit;
+    }
+
+    tr {
+      padding: 0px 5px;
+    }
+
+    td {
+      text-align: center;
+      text-overflow: ellipsis;
+      overflow: hidden;
+      transition: 0.3s ease;
+      transition-property: width, min-width, padding, opacity;
+    }
+
+    th,
+    td {
+      border-right: $table-border-light;
+    }
+
+    thead {
+      max-height: 30px;
+      position: sticky;
+      top: 0;
+      z-index: 50;
+
+      tr {
+        height: 30px;
+      }
+
+      tr:first-child th {
+        border-bottom: 1px solid $table-border-light;
+        border-top: 1px solid $gray-dark;
+        font-weight: bold;
+        color: #000;
+        background-color: #f7f7f7;
+      }
+
+      tr:last-child th {
+        background-color: #fff;
+        border-bottom: $table-border-dark;
+        box-shadow: none;
+        vertical-align: top;
+        div {
+          width: max-content;
+          margin: auto;
+        }
+      }
+
+      th {
+        padding: 7px 5px;
+      }
+    }
+
+    tbody {
+      tr {
+        border-bottom: $table-border-light;
+
+        &.row-odd {
+          background-color: #f7f7f7;
+          &:hover {
+            background-color: #f2f2f2ff;
+          }
+        }
+        &.row-even {
+          background-color: #fff;
+          &:hover {
+            background-color: #f0f0f0;
+          }
+        }
+
+        .contacts-expand {
+          border-radius: 5rem;
+          padding: 0.1rem 0.4rem;
+          margin-left: 0.5rem;
+          background-color: #f2f2f2;
+          color: $justfix-black;
+          border: 1px solid $justfix-black;
+        }
+        .contacts-list {
+          color: $justfix-green;
+        }
+      }
+
+      td {
+        &.col-address {
+          text-align: left;
+          padding: 0.75rem;
+        }
+        &.col-address,
+        &.col-recentcomplaintsbytype,
+        &.col-ownernames,
+        &.col-yearstartedj51,
+        &.col-yearstarted421a,
+        &.col-detail {
+          white-space: unset;
+        }
+      }
+    }
+  }
+}
+
+.pagination {
+  position: sticky;
+  left: 0px;
+  z-index: 50;
+  justify-content: space-between;
+  align-items: stretch;
+  flex-wrap: wrap;
+  padding: 3px;
+  background-color: $wowza-background;
+
+  div {
+    display: block;
+    flex: 1 1;
+    text-align: center;
+  }
+
+  .center {
+    flex: 1.5 1;
+    text-align: center;
+    margin-bottom: 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-around;
+
+    .page-info {
+      display: flex;
+      align-items: center;
+      margin: 3px 10px;
+      white-space: nowrap;
+
+      input {
```

Comment:
just making sure i'm reading this right, this is a custom wow input and not design system, right?

Reply:
Yeah, this is specific to the min/max filter.